gear/gear/database.py

```diff
@@ -21,6 +22,22 @@
 retry_codes = (1213, 2003, 2013)
 
 
+async def _show_warnings(self, conn):
+    if self._result and self._result.has_next:
+        return
+    ws = await conn.show_warnings()
+    if ws is None:
+        return
+    for w in ws:
+        msg = w[-1]
+        last_executed = getattr(self, '_last_executed', None)
+        executed = getattr(self, '_executed', None)
+        warnings.warn(f'{msg} {last_executed} {executed}', Warning, 4)
+
+
+aiomysql.Cursor._show_warnings = _show_warnings
```

Comment:
Will this print out the query with all its parameters? That makes me uneasy from both a security standpoint and generally overriding library methods. Is this very hard to replicate in dev? If it has to be in default then we should remove it quickly after diagnosing the bug.

Reply:
Good call! I didn't even think of that. Hopefully attempt number 2 is cleaner.